### PR TITLE
tests on rawhide should use repo from latest dist build

### PR DESCRIPTION
**Pull request by bgoncalv**

### CODE REVIEW

**Comment by bgoncalv** [comment]
@johnbieren does it make sense?

File: config/Dockerfiles/cloud-image-compose/virt-customize.sh [hunk 1]
```diff
@@ -74,20 +74,20 @@ enabled=1
...
-    koji_repo="f${branch}-build"
     if ! virt-customize --selinux-relabel --memsize 4096 -a ${DOWNLOADED_IMAGE_LOCATION} --run-command "dnf config-manager --set-enable updates-testing updates-testing-debuginfo" ; then
         echo "failure enabling updates-testing repo"
         exit 1
     fi
 else
     virt-customize -a ${DOWNLOADED_IMAGE_LOCATION} --run-command "source /etc/os-release; rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-\$VERSION_ID-fedora"
 fi
+
+koji_repo=$(echo ${DIST_BRANCH}-build | sed -e s'/fc/f/')
```
Comment (ignatenkobrain) [inline config/Dockerfiles/cloud-image-compose/virt-customize.sh]: how does `$DIST_BRANCH` look on rawhide? is that `f30` now? Or is it `master`?

Comment (bgoncalv) [reply config/Dockerfiles/cloud-image-compose/virt-customize.sh]: It should be `fc30`.

Comment (ignatenkobrain) [reply config/Dockerfiles/cloud-image-compose/virt-customize.sh]: In that case, let's merge it and give it a try!

Comment (johnbieren) [reply config/Dockerfiles/cloud-image-compose/virt-customize.sh]: Yep, $DIST_BRANCH should *always* be fcXX where XX is the major release rawhide maps to

**Comment by johnbieren** [comment]
[upstream-merge]

**Pull request merged**